src/cholesky.jl

```diff
@@ -12,48 +12,46 @@ end
 end
 @inline LinearAlgebra._chol!(A::StaticMatrix, ::Type{UpperTriangular}) = (cholesky(A).U, 0)
 
-
-@generated function _cholesky(::Size{(1,1)}, A::StaticMatrix)
-    @assert size(A) == (1,1)
-
-    quote
-        $(Expr(:meta, :inline))
-        T = promote_type(typeof(sqrt(one(eltype(A)))), Float32)
-        similar_type(A,T)((sqrt(A[1]), ))
+@generated function _cholesky(::Size{S}, A::StaticMatrix{M,M}) where {S,M}
+    @assert (M,M) == S
+    M > 24 && return :(_cholesky_large(Size{$S}(), :A))
+    q = Expr(:block)
+    for n in 1:M
+        for m ∈ n:M
```

Comment:
Tiny syntax quibble - `in` vs `∈` ? Let's choose one :)

Reply:
Done.